Bottom: 89c188b93e7fbc543e61884c297bbc1335ac369d
Top:    c272de4e461873d17ab14b2b5df6703e15f6af05
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 23:16:20 +0000

Refresh of add-configure-checks-for-gtkgl

---

diff --git a/configure.ac b/configure.ac
index 77b4509..bee58a6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -406,8 +406,6 @@ Either do not use --enable-gl or enable the gtk HID.])
 	;;
 	esac
 
-# FIXME... what do we need to check for determining we have GL? Is it headers, API ?
-
 	AC_CHECK_HEADERS(GL/gl.h)
 	case $ac_cv_header_GL_gl_h in
 	  no )
